invenio_circulation/transitions/base.py

```diff
@@ -172,7 +172,6 @@ def _date_fields2datetime(self, kwargs):
     def before(self, loan, initial_loan, **kwargs):
         """Validate input, evaluate conditions and raise if failed."""
         loan.update(kwargs)
-        loan.setdefault("transaction_date", arrow.utcnow())
```

Comment:
I would leave it in case the transaction is triggered programmaticaly, not via REST, ping @ntarocco